app/src/main/java/com/palone/planahead/screens/taskEdit/ui/taskTypeSelection/DateField.kt

```diff
@@ -1,36 +1,100 @@
 package com.palone.planahead.screens.taskEdit.ui.taskTypeSelection
 
-import androidx.compose.foundation.clickable
-import androidx.compose.material.icons.Icons
-import androidx.compose.material.icons.filled.CalendarMonth
-import androidx.compose.material3.Icon
-import androidx.compose.material3.OutlinedTextField
-import androidx.compose.material3.Text
+import android.util.Log
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawWithContent
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.palone.planahead.screens.taskEdit.ui.selectors.DateSelector
+import com.palone.planahead.screens.taskEdit.ui.selectors.TimeSelector
+import java.time.Instant
 import java.time.LocalDateTime
-import java.time.format.DateTimeFormatter
+import java.time.ZoneId
 
 @Composable
 fun DateField(
     modifier: Modifier = Modifier,
-    dateAndTime: LocalDateTime,
-    onChooseDateAndTimeEvent: () -> Unit
+    todayDateAndTime: LocalDateTime,
```

Comment:
Does it always have to be today's date/date? What about editing of a task? We want the default to be set to date/time on the task, no?